benchmarks/curl_curl_fuzzer_http/benchmark.yaml

```diff
@@ -21,3 +21,4 @@ unsupported_fuzzers:
   - libfuzzer_dataflow
   - libfuzzer_dataflow_load
   - libfuzzer_dataflow_store
```

Comment:
Can you also add centipede please?

Reply:
ah! yes, that's the original failing one.

Reply:
Thanks for adding `Centipede` : )